lib/widgets.js

```diff
@@ -43,6 +45,57 @@ var input = function (type) {
     };
 };
 
+var isString = function (value) {
+    return typeof value === 'string';
+};
+
+var isArray = Array.isArray;
```

Comment:
since we support down to node 0.4, please use https://npmjs.com/isarray (but only `^1.0.1`) instead.